tests/data/expected/openapi/default_template/body_and_parameters_reuse_model/main.py

```
@@ -0,0 +1,155 @@
+
+
+@app.post('/bar', response_model=None, tags=['bar'])
+def post_bar(request: Request) -> None:
+    """
+    Create a bar
+    """
+    pass
+
+
+@app.post('/convert', response_model=bytes)
+def convert1(format: Optional[str] = 'pdf', request: Request = ...) -> bytes:
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)